### PR TITLE
Feature/print behavior

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**
implementation of https://github.com/IntelPython/daal4py/issues/7

Examples of printing:
low order moments print(result):
```
minimum: array, type=float64, shape=(1, 10)
  [[-9.948613 -9.737347 -9.619284 -9.743521 -9.990568 -9.9973   -9.997706
    -9.897363 -9.794658 -9.861839]]

maximum: array, type=float64, shape=(1, 10)
  [[9.90086  9.869278 9.997498 9.868528 9.404993 9.768626 9.910092 9.896053
    9.647098 9.971762]]

sum: array, type=float64, shape=(1, 10)
  [[ -13.894732   65.933167   30.511676   40.409349  -64.703292  -94.211596
      32.531087   69.566359 -131.328752   42.703151]]

sumSquares: array, type=float64, shape=(1, 10)
  [[7249.157216   6569.17370838 6068.50429546 5977.01040518 6720.27291173
    6209.56635712 6550.82550358 6637.25266512 6995.62512713 7102.79385404]]

sumSquaresCentered: array, type=float64, shape=(1, 10)
  [[7248.19189812 6547.43779583 6063.8494836  5968.84582775 6699.34033176
    6165.18723302 6545.53414547 6613.05527359 6909.38892162 7093.67605851]]

mean: array, type=float64, shape=(1, 10)
  [[-0.06947366  0.32966584  0.15255838  0.20204675 -0.32351646 -0.47105798
     0.16265544  0.34783179 -0.65664376  0.21351576]]

secondOrderRawMoment: array, type=float64, shape=(1, 10)
  [[36.24578608 32.84586854 30.34252148 29.88505203 33.60136456 31.04783179
    32.75412752 33.18626333 34.97812564 35.51396927]]

variance: array, type=float64, shape=(1, 10)
  [[36.42307486 32.90169747 30.47160545 29.99420014 33.66502679 30.98084037
    32.89213138 33.23143354 34.72054734 35.64661336]]

standardDeviation: array, type=float64, shape=(1, 10)
  [[6.03515326 5.73600013 5.52010919 5.4766961  5.80215708 5.56604351
    5.7351662  5.76467116 5.89241439 5.97047849]]

variation: array, type=float64, shape=(1, 10)
  [[-86.86966052  17.39943761  36.18358551  27.10608427 -17.93465804
    -11.81604759  35.25960386  16.5731576   -8.97353291  27.96270695]]
```
linear regression print(train_result):
```
model:
  NumberOfBetas: 11

  NumberOfResponses: 2

  InterceptFlag: True

  Beta: array, type=float64, shape=(2, 11)
    [[ 2.41272553  9.25181248  0.3856898  -6.28711921 -4.12093117  7.77676348
       7.83651727  3.30835195  9.10860594 -6.73231343 -1.83129615]
     [ 2.47020425 -4.97377057  1.17108509  8.94688555 -7.54706096  2.71588871
       6.26894149  7.72190342 -8.53511092  1.71497193  5.05079029]]

  NumberOfFeatures: 10
```

### CODE REVIEW

**Comment by fschlimb** [comment]
@triskadecaepyon @oleksandr-pavlyk @anton-malakhov This implements #7. Please comment on the print behavior.

**Comment by fschlimb** [comment]
If no other comments/suggestions by then, let's merge after 2019.3 has been merged and CI is green.

File: generator/wrapper_gen.py [hunk 1]
```diff
@@ -177,6 +177,25 @@ def my_procid():
...
+        for p in properties:
+            prop = getattr(instance, p, None)
+            if isinstance(prop, np.ndarray):
+                result += p + ': array, type={}, shape={}'.format(prop.dtype,
+                                                                  prop.shape)
+            elif isinstance(prop, dict):
+                result += p + ': dict, len={}'.format(len(prop))
+            else:
+                result += p + ': '
+            value = str(prop).replace('\\n', '\\n  ')
```
Comment (oleksandr-pavlyk) [inline generator/wrapper_gen.py]: Can you use ``os.linesep`` instead of `\\n`, see https://stackoverflow.com/a/8325958/594376

Comment (fschlimb) [reply generator/wrapper_gen.py]: I guess the proper way to do this is the following
```
value = '\n  '.join(str(prop)).splitlines())
```

**Pull request closed without merging**